### PR TITLE
Get customized analysis results with result_type

### PR DESCRIPTION
**Pull request by vnadhan**
# Description

Include a summary of this pull request

New dependencies: (Added to requirements_*.txt)

Fixes # (potential issues that are fixed by this PR)

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue -> bump patch)
- [x] New feature (non-breaking change which adds functionality -> bump minor version)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected -> bump major version)
- [ ] Documentation Update
- [ ] CI/CD workflows Update

## Checklist

- [ ] I have added tests that prove that my fix/feature works
- [x] Linters pass locally and I have followed PEP8 code style
- [ ] New and existing tests pass locally
- [ ] I have updated the documentation if needed
- [ ] I have commented hard-to-understand areas in the code

## Requirements

- [x] I have updated the MVG version

### CODE REVIEW

File: tests/test_api_analyses.py [hunk 1]
```diff
@@ -275,3 +275,55 @@ def test_get_analysis_info(session: MVG, waveform_source_multiaxial_001):
...
+):
+    source_id = waveform_source_with_measurements
+
+    request = session.request_analysis(source_id, "KPIDemo")
+    request_id = request["request_id"]
+    result_type = "compressed"
+    session.wait_for_analyses([request_id])
+
+    with pytest.raises(MVGAPIError) as _:
+        session.get_analysis_results(request_id, result_type=result_type)
```
Comment (tuix) [inline tests/test_api_analyses.py:315]: to make sure that the exception is raised for the right reason it is better to check the returned message, similar to `test_delete_analysis`

File: tests/test_api_analyses.py [hunk 2]
```diff
@@ -275,3 +275,55 @@ def test_get_analysis_info(session: MVG, waveform_source_multiaxial_001):
...
+):
+    source_id = waveform_source_with_measurements
+
+    request = session.request_analysis(source_id, "KPIDemo")
+    request_id = request["request_id"]
+    result_type = "someresultype"
+    session.wait_for_analyses([request_id])
+
+    with pytest.raises(MVGAPIError) as _:
+        session.get_analysis_results(request_id, result_type=result_type)
```
Comment (tuix) [inline tests/test_api_analyses.py:329]: same as previous comment

File: tests/test_api_analyses.py (hunk 1 above)
Comment (vnadhan) [reply tests/test_api_analyses.py:315]: Good point. This will be integrated

**Review by tuix — APPROVED** [review]

File: tests/test_api_analyses.py (hunk 1 above)
Comment (vnadhan) [reply tests/test_api_analyses.py:316]: Resolved in https://github.com/vikinganalytics/mvg/pull/205/commits/665d45dc6e3ee4085dd9bae6fa653fa377853ae3

File: tests/test_api_analyses.py (hunk 2 above)
Comment (vnadhan) [reply tests/test_api_analyses.py:331]: Resolved in https://github.com/vikinganalytics/mvg/pull/205/commits/665d45dc6e3ee4085dd9bae6fa653fa377853ae3

**Pull request merged**